Integration and User password reset (#321)
ovh/terraform-provider-ovh: website/docs/r/cloud_project_database_m3db_user.html.markdown

  group         = "mygroup"
  name          = "johndoe"
}

output "user_password" {
    value = ovh_cloud_project_database_m3db_user.user.password
    sensitive = true
}
```

> To reset password of the user previouly create, update the `password_reset` attribut.

Reviewer: ```suggestion
-> __NOTE__ To reset password of the user previously created, update the `password_reset` attribute.
```

> Use command `terraform refresh` after `terraform apply` to update output with new password.

Reviewer: ```suggestion
Use command `terraform refresh` after executing `terraform apply` to update the output with new password.
```

* `name` - (Required, Forces new resource) Name of the user.

> * `password_reset` - (Optional) Arbitrary string to change to trigger a password update.

Author: no it's for all engines

> 

Reviewer: ```
output "user_password" {
    value = ovh_cloud_project_database_m3db_user.user.password
    sensitive = true
}
```

Reviewer: speak about the `terraform refresh` command to execute after the apply ... :)

* `name` - (Required, Forces new resource) Name of the user.

> * `password_reset` - (Optional) Arbitrary string to change to trigger a password update. Use command `terraform refresh` after `terraform apply` to update output with new password.

Reviewer: ```suggestion
* `password_reset` - (Optional) Arbitrary string to change to trigger a password update. Use command `terraform refresh` after executing `terraform apply` to update the output with new password.
```
